doc/radosgw/lua-scripting.rst

```diff
@@ -0,0 +1,338 @@
+| ``S3Request.HTTP.Parameters``                        | string   |                                                              | no       | no        | no       |
++------------------------------------------------------+----------+--------------------------------------------------------------+----------+-----------+----------+
+| ``S3Request.HTTP.Domain``                            | string   |                                                              | no       | no        | no       |
++------------------------------------------------------+----------+--------------------------------------------------------------+----------+-----------+----------+
+| ``S3Request.HTTP.StorageClass``                      | string   |                                                              | no       | no        | no       |
++------------------------------------------------------+----------+--------------------------------------------------------------+----------+-----------+----------+
+| ``S3Request.Time``                                   | time     | request time                                                 | no       | no        | no       |
++------------------------------------------------------+----------+--------------------------------------------------------------+----------+-----------+----------+
+| ``S3Request.Dialect``                                | string   | "S3" or "Swift"                                              | no       | no        | no       |
++------------------------------------------------------+----------+--------------------------------------------------------------+----------+-----------+----------+
+| ``S3Request.Id``                                     | string   | request Id                                                   | no       | no        | no       |
++------------------------------------------------------+----------+--------------------------------------------------------------+----------+-----------+----------+
+| ``S3Request.TransactionId``                          | string   | transaction Id                                               | no       | no        | no       |
++------------------------------------------------------+----------+--------------------------------------------------------------+----------+-----------+----------+
+| ``S3Request.Tags``                                   | table    | object tags map                                              | yes      | no        | no       |
++------------------------------------------------------+----------+--------------------------------------------------------------+----------+-----------+----------+
+
+S3 Request Functions
+--------------------
+Operations Log
+~~~~~~~~~~~~~~
+The ``S3Request.Log()`` function prints the requests into the operations log. This function has no parameters and return zero on success and an error code if it fails. 
```

Comment:
This function has no parameters. It returns `0` for success and an error code if it fails.